Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs

```diff
@@ -26,218 +23,143 @@ public class StorageAdapterTest
         private const string DatabasePath = "databaseAtRoot.db";
 
         [TestInitialize]
-        public void InitializeStorageTest()
+        public void TestInitialize()
         {
-            _adapter = new StorageAdapter();
-        }
-
-        [TestMethod]
-        public void InitializeStorageCreatesStorageDirectory()
-        {
-            var adapter = new StorageAdapter();
-            Microsoft.AppCenter.Utils.Constants.AppCenterFilesDirectoryPath = Environment.CurrentDirectory;
             Microsoft.AppCenter.Utils.Constants.AppCenterDatabasePath = DatabasePath;
             try
             {
-                adapter.Initialize(DatabasePath);
-            }
-            catch
-            {
-                // Handle exception, database is not created with Mock.
-            }
-            Assert.IsTrue(System.IO.File.Exists(DatabasePath));
-        }
-
-        [TestMethod]
-        public void FaildToOpenDatabaseWhenNameWrong()
-        {
-            var adapter = new StorageAdapter();
-            try
-            {
-                adapter.Initialize("test://test.txt");
-                Assert.Fail("Should have thrown exception");
+                System.IO.File.Delete(DatabasePath);
             }
             catch (Exception e)
             {
-                Assert.IsTrue(e.Message.Contains("Failed to open database connection"));
+                Console.WriteLine("Fail! {0}", e.Message);
```

Comment:
Does logging work on those unit tests?

Reply:
Why removing assert?

Reply:
Shouldn't it be something like `Assert.Fail`?

Reply:
Commited by mistake. That shouldn't be here at all - used it for debugging